### PR TITLE
 social media icons have been fixed.

### PR DESCRIPTION
**Pull request by Dilekcn**

### CODE REVIEW

File: client/src/pages/Home/FooterIcon/FooterIcon.css [hunk 1]
```diff
@@ -1,42 +1,39 @@
...
 }
 .footericon_text {
-    font-size: 15px;
-    font-family: "Sans Regular";
-    color: var(--white-color);
-    margin: 20px;
+	font-size: 15px;
+	font-family: "DMSans-Regular";
+	color: var(--white-color);
+	margin: 20px;
```
Comment (khaledkz) [inline client/src/pages/Home/FooterIcon/FooterIcon.css:34]: can you please fix indentation

File: client/src/pages/Terms&Condition/TermsConditions.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function TermsConditions() {
+  return <div>
+HELLO FROM TERMS & cONDITIONS
```
Comment (khaledkz) [inline client/src/pages/Terms&Condition/TermsConditions.js:5]: can you please add space here also (cONDITIONS) looks need to be all uppercase

File: client/src/pages/Home/FooterIcon/FooterIcon.css (hunk 1 above)
Comment (Dilekcn) [reply client/src/pages/Home/FooterIcon/FooterIcon.css:34]: Hi @khaledkz 
The indentation fixed

File: client/src/pages/Terms&Condition/TermsConditions.js (hunk 2 above)
Comment (Dilekcn) [reply client/src/pages/Terms&Condition/TermsConditions.js:5]: It's just to a placeholder. will delete later

**Review by filkovsp — APPROVED** [review]

**Pull request merged**